patch/llpcPatchEntryPointMutate.cpp

```diff
@@ -516,12 +517,22 @@ FunctionType* PatchEntryPointMutate::GenerateEntryPointType(
                 availUserDataCount -= 1;
             }
 
-            if (builtInUsage.vs.baseVertex || builtInUsage.vs.baseInstance)
+	    // on GFX9+, LS and HS are merged into a single shader.
```

Comment:
// on GFX9+, LS and HS are merged into a single shader.
->
// NOTE:  On GFX9+, Vertex shader (LS) and tessellation control shader (HS) are merged into a single shader.